tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -84,8 +86,10 @@ elif [ $BUILD_OS = "yocto" ]; then
         --docker-build-args="--build-arg TOOL_CHAIN=$TOOL_CHAIN_SCRIPT --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER}" \
         --dockerfile $DOCKER_FILE --context .
 elif [ $BUILD_DEVICE = "gpu" ]; then
-        #This code path is only for training. Inferecing pipeline uses CentOS
+        # This code path is only for training. Inferecing pipeline uses CentOS
         IMAGE="$BUILD_OS-gpu_training"
+        # Training did not need build shared library
```

Comment:
Does mean training APIs are only available in C++ and python, not the other languages?

Reply:
Currently yes, I think even C++ we don't have public APIs.